src/openms/source/FORMAT/CsvFile.cpp

```diff
@@ -64,6 +64,11 @@ namespace OpenMS
     load(filename, true, first_n);
   }
 
+  void CsvFile::fstore(const String& filename)
+  {
+    store(filename);
```

Comment:
Wow really?  That was it?  I am slightly embarrased I did not just go ahead and do this a while back...